src/app/app.component.ts

```diff
@@ -1,16 +1,19 @@
 import { Component } from '@angular/core';
 import { FormBuilder } from '@angular/forms';
+import {MatStepper} from '@angular/material/stepper';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
 })
 export class AppComponent {
+  step = 1;
```

Comment:
typescript understands that `step` is public as default but in order to get consistent we still should leave `public` keyword

Reply:
@hoangngk Updated